### ISSUE TITLE
Style/StructInheritance auto-correct can result in an empty block lint violation if the class is empty.

### ISSUE THREAD

**Issue by snood1205** [rubocop/rubocop#9627]
The `Style/StructInheritance` auto-correct treats all `class _ < Struct.new` inheritance to `Struct.new do; end` instead of just `Struct.new`

--------

## Expected behavior
If there a struct defined in the following format with an empty class body, as portrayed below:
```ruby
class Foo < Struct.new(:bar, :baz)
end
```
It should be corrected to 
```ruby
Foo = Struct.new(:bar, :baz)
```
## Actual behavior
The class mentioned above is corrected to
```ruby
Foo = Struct.new(:bar, :baz) do
end
```
This will cause another rubocop violation for an empty block upon running rubocop again.

## Steps to reproduce the problem
Any `Struct` declared in `class` syntax with an empty body will cause this problem

## RuboCop version

```
$ bundle exec rubocop -V
1.11.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 2.7.1 x86_64-darwin19)
  - rubocop-rspec 2.2.0
```
